server/src/util.js

```diff
@@ -0,0 +1,54 @@
+const YAML = require('yamljs');
+const path = require('path');
+
+/**
+ * Parses the input specification
+ * @param {object} file - The input file received as form-data
+ * @returns {object} The JSON for the input specification
+ */
+function parseInputFile(file) {
+  const fileExt = path.parse(file.originalname).ext;
+  var origFile = Buffer.from(file.buffer).toString();
+
+  if(fileExt === '.json') {
+    origFile = JSON.parse(origFile);
+    return origFile;
+  }
+  else if(fileExt === '.yaml') {
+    origFile = YAML.parse(origFile);
+    return origFile;
+  }
+  else {
+    return null;
+  }
+}
+
+/**
+ * Sends response based on the requirement
+ * @param {object} result - The output file received from lucy's converter
```

Comment:
This isn't used now, right? If that's the case can we remove this? It would simplify the code. If it's needed at a later date, we should be able to send it as a return object.

Reply:
This is being used within this function as it is the output that is coming in from the lucy's converter.